api/src/pcapi/alembic/versions/20230502T154422_c292ae40e99f_create_table_collectiverequest.py

```diff
@@ -0,0 +1,68 @@
+"""create_table_collectiveRequest
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "c292ae40e99f"
+down_revision = "0e0fe7986d01"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "collective_request",
+        sa.Column("id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("email", sa.String(length=120), nullable=False),
+        sa.Column("phoneNumber", sa.String(length=30), nullable=True),
+        sa.Column("requestedDate", sa.DateTime(), nullable=True),
+        sa.Column("requestedTime", sa.String(length=10), nullable=True),
+        sa.Column("totalStudent", sa.Integer(), nullable=True),
+        sa.Column("totalResponsable", sa.Integer(), nullable=True),
+        sa.Column("comment", sa.Text(), nullable=False),
+        sa.Column("collectiveOfferTemplateId", sa.BigInteger(), nullable=False),
+        sa.Column("educationalRedactorId", sa.BigInteger(), nullable=False),
+        sa.Column("educationalInstitutionId", sa.BigInteger(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["collectiveOfferTemplateId"],
+            ["collective_offer_template.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["educationalInstitutionId"],
+            ["educational_institution.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["educationalRedactorId"],
+            ["educational_redactor.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("email"),
+    )
+    op.create_index(
+        op.f("ix_collective_request_collectiveOfferTemplateId"),
+        "collective_request",
+        ["collectiveOfferTemplateId"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_collective_request_educationalInstitutionId"),
+        "collective_request",
+        ["educationalInstitutionId"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_collective_request_educationalRedactorId"),
+        "collective_request",
+        ["educationalRedactorId"],
+        unique=False,
+    )
+
+
+def downgrade() -> None:
+    op.drop_index(op.f("ix_collective_request_educationalRedactorId"), table_name="collective_request")
+    op.drop_index(op.f("ix_collective_request_educationalInstitutionId"), table_name="collective_request")
+    op.drop_index(op.f("ix_collective_request_collectiveOfferTemplateId"), table_name="collective_request")
```

Comment:
Tu peux supprimer les "drop index" : lors de la suppression de la table, PostgreSQL supprimera automatiquement ces index.

```suggestion
    op.drop_index(op.f("ix_collective_request_educationalRedactorId"), table_name="collective_request")
    op.drop_index(op.f("ix_collective_request_educationalInstitutionId"), table_name="collective_request")
    op.drop_index(op.f("ix_collective_request_collectiveOfferTemplateId"), table_name="collective_request")
```